dontusenativedialog.patch

```diff
@@ -0,0 +1,155 @@
+From 74b386653a0ab95c2a0f73dda6af761d367d6afb Mon Sep 17 00:00:00 2001
```

Comment:
Has this patch been submitted upstream?

Reply:
Yes, but they won't allow it. The developer don't want to maintain any kind of flatpak runtime changes. That's the only reason I am using an external patch that I am applying on the flatpak build.

Reply:
Do you have an URL to said discussion?

Reply:
OK, it's at https://github.com/puddletag/puddletag/issues/735. I would recommend following up on this with @tinywrkb suggestion to make this dynamic based on `/.flatpak-info` presence.

Reply:
yes: https://github.com/puddletag/puddletag/issues/735